src/pkg/scan/postprocessors/report_converters.go

```diff
@@ -105,6 +110,11 @@ func (c *nativeToRelationalSchemaConverter) toSchema(ctx context.Context, report
 		return err
 	}
 
+	// set a upper limit of vulnerability number to 60000 because of the limit of postgresql query parameters count limit is 65535
```

Comment:
Is it reasonable to limit this place to 60,000? Based on PostgreSQL’s limitations, shouldn’t we process all records in pages or batches rather than limiting the quantity here?